app/uk/gov/hmrc/fhregistrationfrontend/views/forms/trading_name.scala.html

```diff
@@ -41,39 +40,89 @@
   Messages("fh.tradingName.confirm.label")
 }
 
-@main_template(title = titlePrefix + title, scriptElem = Some(pageScripts)) {
+@backLink(navigation: Navigation) = @{
```

Comment:
Will remove this code later while rebasing to use the file back.scala.html. Didn't want to change all those Back to backOld in this PR. That will be included when I rebase it with previous PR of Vat form